grab-image.php

```diff
@@ -330,16 +338,12 @@ function grab_image_post() {
                 . "<img class=\"alignnone size-full wp-image-{$id}\" src=\"{$src}\" alt=\"{$post->post_title}\" />"
                 . "</a>";
 
-            // set post thumbnail
-            if ($thumb == 0) {
-                set_post_thumbnail($post, $attachmentId);
-                $thumb = 1;
-            }
-
             $exist = ($exist ? '' : 'none-exist');
             echo "success ; {$url} ; {$attachmentId} ; {$exist} <br/>";
```

Comment:
I know this isn't new, but the success message is rather confusing.

`echo "<a href=\"{.get_edit_post_link($attachmentId).}">{$url}</a> was successfully uploaded. It {$exist} already.<br/>";`